src/main/java/servlets/module/challenge/NoSqlInjection1.java

```diff
@@ -3,45 +3,42 @@
 import java.io.IOException;
 import java.io.PrintWriter;
 
-import com.mongodb.BasicDBObject;
-import com.mongodb.DB;
-import com.mongodb.DBObject;
-import com.mongodb.DBCollection;
-import com.mongodb.DBCursor;
-import com.mongodb.MongoClient;
-import com.mongodb.MongoCredential;
-import com.mongodb.MongoException;
+import com.mongodb.*;
```

Comment:
Isn't it bad practice to import everything when we only need a subset of com.mondodb?

Reply:
Not doing this anymore (Intellij seems to do this automatically)